client/src/ui/modules/chat/Chat.tsx

```
@@ -228,32 +254,56 @@
                   .filter((tab) => ContractAddress(tab.address) !== ContractAddress(account.address))
                   .map((player, index) => (
-                    <SelectItem className="flex justify-between" key={index} value={toHexString(player.address)}>
+                    <SelectItem
+                      className="flex justify-between"
+                      style={{ color: PASTEL_BLUE }}
+                      key={index}
+                      value={toHexString(player.address)}
+                    >
                       {player.addressName}
                     </SelectItem>
                   ))}
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)